game/core/src/me/lihq/game/living/controller/PlayerController.java

```diff
@@ -45,6 +45,18 @@ public boolean keyDown(int keycode)
             return true;
         }
 
+        if (keycode == Input.Keys.J)
```

Comment:
If we could add a F key the disables or enables the debug mode that would be great for the presentation